src/games/strategy/ui/SwingComponents.java

```diff
@@ -403,9 +403,7 @@ protected void done() {
           promise.completeExceptionally(e.getCause());
         } catch (final InterruptedException e) {
           promise.completeExceptionally(e);
-          // TODO: consider if re-interrupting the thread is the right thing to do when we can't throw
-          // InterruptedException (see https://www.ibm.com/developerworks/library/j-jtp05236/)
-          ClientLogger.logQuietly(e);
```

Comment:
Remove the client logger. Not much value in logging an interrupted exception. In addition to that, the UI window is closing, so we are also logging into a black hole.